test/unit/DefinitionList.test.js

```diff
@@ -18,44 +19,43 @@ describe('DefinitionList', () => {
 
   it('should display a definition list of detail elements', () => {
     expect(component.find('dl')).to.have.length(1);
-    expect(component.find('dt')).to.have.length(6);
-    expect(component.find('dd')).to.have.length(6);
+    expect(component.find('dt')).to.have.length(5);
+    expect(component.find('dd')).to.have.length(5);
     const terms = component.find('dt');
     expect(terms.nodes[0].props.children).to.equal(labels.title);
-    expect(terms.nodes[1].props.children).to.equal(labels.rights_stmt);
-    expect(terms.nodes[2].props.children).to.equal(labels.language);
-    expect(terms.nodes[3].props.children).to.equal(labels.instances);
-    expect(terms.nodes[4].props.children).to.equal(labels.entities);
-    expect(terms.nodes[5].props.children).to.equal(labels.subjects);
+    expect(terms.nodes[1].props.children).to.equal(labels.language);
+    expect(terms.nodes[2].props.children).to.equal(labels.agents);
+    expect(terms.nodes[3].props.children).to.equal(labels.subjects);
+    expect(terms.nodes[4].props.children).to.equal(labels.instances);
   });
 
   it('should have a table of Items', () => {
     expect(component.find('table')).to.have.length(1);
     expect(component.find('thead')).to.have.length(1);
     expect(component.find('tbody')).to.have.length(1);
-    expect(component.find('tbody tr')).to.have.length(1);
-    expect(component.find('tbody tr td')).to.have.length(4);
+    expect(component.find('tbody tr')).to.have.length(182);
+    expect(component.find('tbody tr td')).to.have.length(728);
   });
 
   it('should have a list of Subjects', () => {
     const subjects = component.find('ul');
-    expect(subjects.nodes[1].props.children).to.have.length(9);
+    expect(subjects.nodes[2].props.children).to.have.length(92);
   });
 
   it('should have a list of Authors', () => {
     const authors = component.find('ul');
-    expect(authors.nodes[0].props.children).to.have.length(2);
+    expect(authors.nodes[0].props.children).to.have.length(1);
   });
 
   describe('EBookList', () => {
     before(() => {
-      const ebooks = detail.work.instances[0].items;
+      const ebooks = detail.instances[8].items;
       component = shallow(<EBookList ebooks={ebooks} />);
     });
     it('should have a list of two links', () => {
```

Comment:
Need to change the text here to reflect what's being tested -- four links instead of two.

Reply:
done